e2e-tests/native/wdio-ios.conf.ts

```diff
@@ -10,6 +10,7 @@ const iosCapabilities: Capabilities.DesiredCapabilities = {
   'appium:orientation': 'PORTRAIT',
   'appium:automationName': 'XCUITest',
   'appium:language': 'EN',
+  'appium:fullReset': true,
```

Comment:
A second installation of an app in iOS doesn't reset the permissions; without this full reset, only one in the four tests would request permission to send notifications.

Reply:
:wrench: Please add a comment in the code to explain this :)